google/earthenterprise
# Fix user home directory permissions

Fixes #1913 

**Verification Steps**
1. Build and install for Ubuntu 16
2. Verify the /opt/google/.users/* directories are owned by each user (gepguser, geapacheuser, gefusionuser) and assigned to the group gegroup
3. Build RPMs for el7 and install
4. Verify the /opt/google/.users/* directories are owned by each user (gepguser, geapacheuser, gefusionuser) and assigned to the group gegroup
5. For users gepguser and geapacheuser, verify that the .profile file also has the same ownership info
